### DISCUSSION TITLE
CLI option --no-live-reload does not work

### DISCUSSION THREAD

**Discussion by deleonio** [webpack/webpack#14419] [Q&A]
# Bug report

Should: https://webpack.js.org/configuration/dev-server/#devserverlivereload

**What is the current behavior?**
![image](https://user-images.githubusercontent.com/6279703/136190220-269f4b1d-af78-4a47-8c2d-d1a96819a884.png)

**If the current behavior is a bug, please provide the steps to reproduce.**

**What is the expected behavior?**

**Other relevant information:**
webpack version:  v5
Node.js version: 16
Operating System: win / Mac
Additional tools:

**Comment by alexander-akait** [comment]
Run npm `ls webpack` and `npm ls webpack-dev-server`

**Reply by alexander-akait** [reply]
And please put string of command, not screenshot

**Comment by deleonio** [comment]
Thanks ....

 @leanup/stack-webpack@1.2.0-rc.87
  `-- webpack-dev-server@3.11.2

We should upgrade webpack-dev-server first.

**Comment by deleonio** [comment]
But it's done: https://github.com/leanupjs/leanup/blob/cc7e1523cd3991b8a36e6c62cad17f55aa2b1060/packages/stack/webpack/package.json#L46

Why he used the webpack-dev-server v3?!

**Reply by alexander-akait** [reply]
You need check it locally, try to regenerate lock file and run again `npm ls webpack-dev-server`
